docs/how-to-guides/otel-collector/otel-collector-scrubbing.md

```diff
@@ -0,0 +1,148 @@
+
+* **After:** `user.comment` = "My email is `***`, please contact me."
+
+ ---
+
+### Scenario 3: Conditional Scrubbing with Logic
+
+The [transform processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/transformprocessor/README.md) has a powerful query language called OTTL which lets you apply conditional logic. For example, here's how to scrub the `credit_card_number` attribute, but **only** if the transaction failed, i.e. `http.status_code` is 500 or greater.
+
+Collector `config.yaml` snippet:
+```yaml
+processors:
+  transform:
+    trace_statements:
+      - set(span.attributes["credit_card_number"], "[REDACTED]") where span.attributes["http.status_code"] >= 500
+```
+
+---
+
+## Complete Example
+
+To use these processors, you need to add them to a service pipeline in your collector configuration. The data will flow through them in the order you specify.
+
+Here is a complete `config.yaml` showing how you might chain these processors together:
+
+```yaml
+# 1. RECEIVERS: How the collector ingests data
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: "0.0.0.0:4317"
+      http:
+        endpoint: "0.0.0.0:4318"
+
+# 2. PROCESSORS: How we scrub and modify the data
+processors:
+  # First, do simple key-based scrubbing/removal.
+  attributes:
+    - key: session_id
+      action: update
+      value: "[Scrubbed due to session_id]"
+    - key: user_token
+      action: update
+      value: "[Scrubbed due to user_token]"
+
+  # Next, find and mask any PII values we missed.
+  redaction:
+    allow_all_keys: true
+    blocked_values:
+     - '[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}'
+
+  # Finally, apply complex conditional rules.
+  transform:
+    trace_statements:
+      - set(span.attributes["credit_card_number"], "[REDACTED]") where span.attributes["http.status_code"] >= 500
+
+# 3. EXPORTERS: Where the scrubbed data is sent
+exporters:
+  debug:
+  otlphttp:
+    # Configure the US / EU endpoint for Logfire.
+    # - US: https://logfire-us.pydantic.dev
+    # - EU: https://logfire-eu.pydantic.dev
+    endpoint: "https://logfire-eu.pydantic.dev"
+    headers:
+      Authorization: "Bearer ${env:LOGFIRE_TOKEN}"
+
+# 4. SERVICE: The pipeline that connects everything
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [attributes, redaction, transform]
+      exporters: [otlphttp, debug]
+    logs:
+      receivers: [otlp]
+      processors: [attributes, redaction]
+      exporters: [otlphttp, debug]
+```
+
+Now you should have a clearer sense of what's possible using the OpenTelemetry Collector processors for data scrubbing.
+
+For more details, practical examples, and a complete reference of available processors, take a  look at [OpenTelemtry Collector Processors documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor).
```

Comment:
This isn't a link to documentation. It's a list of folders with unhelpful names without spaces. And most of these aren't related to scrubbing.